akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -57,12 +57,12 @@ abstract class JournalSpec(config: Config) extends PluginSpec(config) {
 
   def writeMessages(fromSnr: Int, toSnr: Int, pid: String, sender: ActorRef, writerUuid: String): Unit = {
     val msgs =
-      if (supportsAtomicPersistAllOfSeveralEvents)
-        (fromSnr to toSnr).map { i ⇒
-          AtomicWrite(PersistentRepr(payload = s"a-$i", sequenceNr = i, persistenceId = pid, sender = sender,
-            writerUuid = writerUuid))
-        }
-      else
+      if (supportsAtomicPersistAllOfSeveralEvents) {
+        List(AtomicWrite((fromSnr to toSnr) map { i ⇒
+          PersistentRepr(payload = s"a-$i", sequenceNr = i, persistenceId = pid, sender = sender,
+            writerUuid = writerUuid)
+        }))
+      } else {
         (fromSnr to toSnr - 1).map { i ⇒
           if (i == toSnr - 1)
             AtomicWrite(List(
```

Comment:
yes, it is `Seq of AtomicWrite of Seq`... weird, is it an expected behaviour to send 
`Seq(AtomicWrite(samePersistenceId, xs), AtomicWrite(samePersistenceId, ys))` instead of 
`Seq(AtomicWrite(samePersistenceId, xs ++ yx))`